src/utils.ts

```diff
@@ -198,7 +198,7 @@ export async function mesonRootDirs(): Promise<string[]> {
     let hasMesonFile: boolean = false;
     let subdirs: vscode.Uri[] = [];
     for (const [name, type] of await vscode.workspace.fs.readDirectory(d)) {
-      if (type === vscode.FileType.File && name == "meson.build") {
+      if ((type === vscode.FileType.File || type === vscode.FileType.SymbolicLink) && name == "meson.build") {
```

Comment:
```suggestion
      if ((type === vscode.FileType.File || type === (vscode.FileType.File | vscode.FileType.SymbolicLink)) && name == "meson.build") {
```
AFAICT this is the correct check. Based on [this sample](https://github.com/microsoft/vscode-extension-samples/blob/main/nodefs-provider-sample/src/extension.ts#L267), and the debugger showing type is 65 for symlinked files.

Reply:
Makes sense. Can you submit a PR? I'll cook up a release tomorrow. Sorry!